api.planx.uk/saveAndReturn/utils.js

```diff
@@ -108,6 +108,7 @@ const sendSingleApplicationEmail = async (template, email, sessionId) => {
       reference: null,
       emailReplyToId: team.notifyPersonalisation.emailReplyToId,
     };
+    if (!session.has_user_saved) await setupEmailEventTriggers(sessionId);
```

Comment:
a little confused reading this line - should the logic here be to only setup email triggers if `session.has_user_saved` is TRUE ?

Reply:
Yeah I'll add a comment here - it is correct but reads weirdly. Might be better to rename the function.

If the user has not yet saved, setup the emailEventTriggers. The `has_user_saved` value gets toggled within this function, which is what Hasura is listening to to set them up.